### PR TITLE
Newsletters: add 230 (2022-12-14)

### PR DESCRIPTION
**Pull request by harding**
- [x] Update releases/RCs and add topic links @harding 
- [x] Client/Services section @bitschmidty 
- [x] Stack Exchange section @bitschmidty 
- [x] Bitcoin Core 25934 @Xekyo 
- [x] LDK 1835 @adamjonas 

Last regular newsletter of the year!

### CODE REVIEW

File: _posts/en/newsletters/2022-12-14-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Factory-optimized LN protocol proposal:** John Law [posted][law
+  factory] to the Lightning-Dev mailing list the description of a
+  protocol optimized for creating [channel factories][topic channel
+  factories].  Channel factories allow multiple users to trustlessly
+  open multiple channels between pairs of users with only a single
+  transaction onchain.  For example 20 users could cooperate to create
```
Comment (glozow) [inline _posts/en/newsletters/2022-12-14-newsletter.md:26]:
```suggestion
  transaction onchain.  For example, 20 users could cooperate to create
```

**Review by glozow — COMMENTED** [review]
Nice summaries!

File: _posts/en/newsletters/2022-12-14-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday (TODO: check for Bitcoin Core 24.0.1)
```
Comment (fanquake) [inline _posts/en/newsletters/2022-12-14-newsletter.md:199]: The binaries are now available, and the website should be updated to reflect this release today.

File: _posts/en/newsletters/2022-12-14-newsletter.md [hunk 3]
```diff
@@ -188,7 +188,40 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [Why is connecting to the Bitcoin network exclusively over Tor considered a bad practice?]({{bse}}116146)
+  Several answers explain that due to the lower cost of being able to generate many
+  Tor addresses as compared to IPv4 and IPv6 addresses, a Bitcoin node operator
+  exclusively using the Tor network could more easily be [eclipse attacked][topic eclipse
+  attacks] when compared to operating on only on clearnet or with a
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-12-14-newsletter.md:195]:
```suggestion
  attacks] when compared to operating only on clearnet or with a
```

**Review by Xekyo — COMMENTED** [review]
Very nice newsletter

File: _posts/en/newsletters/2022-12-14-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+  explanation for his project, [CircuitBreaker][].  This program,
+  designed to be compatible with LND, enforces limits on the number of
+  pending payments ([HTLCs][topic htlc]) the local node will forward on
+  behalf of each of its peers.  For example, consider the worst case
+  HTLC jamming attack:
+
+    ![Illustration of two different jamming attacks](/img/posts/2020-12-ln-jamming-attacks.png)
+
+    With the current LN protocol, Alice is fundamentally limited to
+    concurrently forwarding a maximum of 483 pending HTLCs.  If she
```
Comment (jonatack) [inline _posts/en/newsletters/2022-12-14-newsletter.md:93]: Perhaps link `483 pending HTCLs` to https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#rationale-7

**Review by jonatack — COMMENTED** [review]
A few comments in addition to the excellent feedback already posted.

**Comment by harding** [comment]
@sipa 

> Is https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-December/021271.html worth mentioning (libsecp256k1 version 0.2.0 released)?

Absolutely!  Congrats on the release!  (For reference, it's our policy to announce all major releases, and most minor releases, for all the software listed in our regular *notable code and documentation changes* section, which includes libsecp256k1.)

**Comment by harding** [comment]
Made all suggested edits (thanks @glozow @fanquake @Xekyo @sipa @jonatack !), reviewed other contributions (thanks @Xekyo @adamjonas and 2x @bitschmidty!), updated releases/RCs, and added topic links.  Thanks everyone for another great newsletter and another great year of regular newsletters.

If anyone reading this wants to help do an early review of the year-end newsletter, please send me an email or a message on IRC.

**Comment by bitschmidty** [comment]
ACK on @harding's final edits, rebased and squashed. Thanks @harding @adamjonas @Xekyo @fanquake @glozow @jonatack for your contributions this week!

**Pull request merged**